cfme/tests/cli/test_appliance_log_rotate.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+
+import dateutil
+import pytest
+
+LOGS = ["/var/www/miq/vmdb/log/evm.log",
+        "/var/opt/rh/rh-postgresql95/lib/pgsql/data/pg_log/postgresql.log"]
+IDS = ['evm', 'postgres']
+
+
+def advance_appliance_date_by_day(appliance):
+    """Advance date on the appliance by one day."""
+    txt_date = appliance.ssh_client.run_command('date --rfc-3339=ns').output
+    appliance_date = dateutil.parser.parse(txt_date)
+    td = datetime.timedelta(days=1)
+    advanced_txt_date = (appliance_date + td).strftime("%Y-%m-%d %H:%M:%S%z")
+    appliance.ssh_client.run_command("date -s '{}'".format(advanced_txt_date))
+
+
+@pytest.mark.parametrize('log_path', LOGS, ids=IDS)
+def test_appliance_log_rotate(temp_appliance_preconfig_funcscope, log_path):
+    ''' Checks whether the log is logrotated daily. '''
+    appliance = temp_appliance_preconfig_funcscope
+    appliance.ssh_client.run_command("/etc/cron.daily/logrotate")
```

Comment:
@jaryn What do you think about checking the return code here  and in line 33 below, if the ssh command fails the rest of the test fails.